src/templates/sitemap.xml

```diff
@@ -79,7 +79,7 @@
 
     <url>
         <loc>
-            https://almanac.httparchive.org/en/2019/css
+            https://almanac.httparchive.org/en/2019/caching
```

Comment:
@bazzadp why does the sitemap keep churning like this? we're not adding anything new.

Reply:
Nevermind, I see you're already on it.